README.md

```
@@ -1,126 +1,221 @@
+## 🔧 기술 스택
+
+### 프론트엔드
+- **프레임워크**: Next.js 13 (App Router)
+- **언어**: TypeScript 5.x
+- **상태 관리**: Zustand 4.x
+- **스타일링**: SCSS (Sass), classNames
+- **HTTP 클라이언트**: Axios
+- **에디터**: TipTap
+- **코드 품질**: ESLint, Prettier
+
+### 개발 도구
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)